cli/setup.py

```diff
@@ -51,6 +51,12 @@ def readme():
     # This does not impact any functionality. Also next connexion release will lift this limitation
     # and we will be able to upgrade.
     "connexion==2.7.0",
+    # pin to connexion transitive dependencies
```

Comment:
could you add a note that this can be removed once we upgrade to a connexion version that includes such constraints?

Reply:
Added.